src/Symfony/Component/HttpKernel/Tests/KernelTest.php

```diff
@@ -400,7 +400,7 @@ public function testLocateResourceOnDirectories()
         $kernel
             ->expects($this->exactly(2))
             ->method('getBundle')
-            ->willReturn($this->getBundle(__DIR__.'/Fixtures/Bundle1Bundle', null, null, 'Bundle1Bundle'))
+            ->willReturn($this->getBundle(__DIR__ . '/Fixtures/Bundle1Bundle', null, 'Bundle1Bundle'))
```

Comment:
needs to be reverted

Reply:
Forgot to make relevant fabbot errors disappear. It should be good now!